sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.2.0-beta.4 (Unreleased)
+## 1.2.0 (Unreleased)
```

Comment:
If you are going to release a new version, don't forget to update `CHANGELOG.md`. See [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges) for details.

Reply:
1.2.0 is the version I am planning to release, last release was 1.1.1